Source/Mosa.DeviceSystem/IACPI.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
+
+namespace Mosa.DeviceSystem
+{
+	/// <summary>
+	/// Interface to ACPI
+	/// </summary>
+	public interface IACPI
+	{
+		BaseIOPortWrite ResetAddress { get; set; }
+		BaseIOPortWrite PM1aControlBlock { get; set; }
```

Comment:
Device is giving up control of its resource. Need to think about this more.

Reply:
How is it? It's contained inside the driver, it's the interface for it. Do you mean that it's because the IO ports are used inside PCService?